core/src/test/scala/com/rallyhealth/vapors/factfilter/evaluator/ConditionExprSpec.scala

```diff
@@ -0,0 +1,36 @@
+package com.rallyhealth.vapors.factfilter.evaluator
+
+import com.rallyhealth.vapors.factfilter.Example.{FactTypes, JoeSchmoe}
+import com.rallyhealth.vapors.factfilter.data.{Evidence, FactSet}
+import com.rallyhealth.vapors.factfilter.dsl.CaptureP.unit._
+import org.scalatest.wordspec.AnyWordSpec
+import com.rallyhealth.vapors.factfilter.dsl.ExprDsl._
+
+class ConditionExprSpec extends AnyWordSpec {
+
+  "Expr.OutputWithinSet" should {
+
+    "find an asthma tag in a set that contains it" in {
+      val q = withFactsOfType(FactTypes.Tag).where {
+        _.exists {
+          _.value.in(Set("asthma", "diabetes"))
+        }
+      }
+      val result = eval(JoeSchmoe.factTable)(q)
+      assert(result.output.value)
+      assertResult(Evidence(JoeSchmoe.asthmaTag))(result.output.evidence)
+    }
+
+    "not find an asthma tag in a set that does not contain it" in {
+      val q = withFactsOfType(FactTypes.Tag).where {
+        _.exists {
+          _.value.in(Set("diabetes"))
+        }
+      }
+      val result = eval(JoeSchmoe.factTable)(q)
+      assert(!result.output.value)
+      // TODO: Should this contain the tags that did not match?
```

Comment:
I'm guessing not. I'm trying to think of a use case for that.

Reply:
The only problem is... an answer without evidence is equivalent to no answer. So if I don't have evidence for the existence of a tag, I don't have evidence of the absence of a tag... That is just how it is.

I'm just not sure how this will work with `not(in(Set()))`. If I have some tags and none of the tag facts are in the set, shouldn't the evidence that the tags are "not in the Set" be the tags that are not in the Set?

Reply:
Maybe I can make a special `not in` operator by using a boolean flag on the `SetContains` operator? Then when the Expr is negated, I can negate the contained node.

Reply:
I think that this is closer--there needs to be a way to represent the "not in " test (and supporting evidence)>

Reply:
I'm gonna create a separate ticket to return to this issue.